### ISSUE TITLE
Update race categorization

### ISSUE THREAD

**Issue by chynnakeys** [cfpb/hmda-platform#2734]
An update needs to be made to the race categorization. 

When 2 racial designations are present, one being minority and the other white, regardless of if the minority race is reported in Applicant Race 1: or Applicant Race: 2, the applicant should still be categorized as a minority. 

For example, if applicant A uses codes Applicant Race : 1 = 5 and Applicant Race: 2 = 22, they would still be categorized as Asian. The same case would happen in the inverse, where applicant A uses codes Applicant Race : 1 = 22 and Applicant Race: 2 = 5. They would still be categorized as Asian. 

The updated categorization can e found here: https://github.com/cfpb/hmda-platform/wiki/Race-Categorization-2018 and the changes begin in the 5th **IF** under the Minority Race section.

**Comment by PatrickHSI** [comment]
Quick question about this this @chynnakeys   in the updated logic in the github wiki the following is listed 

```
IF

Applicant Race: 1 = Black or African American (3) or White (5)

AND

Applicant Race: 2 is White (5) or Black or African American (3)

THEN

Categorize as Black or African American

```

But should the logic reflect the following :

```
IF

Applicant Race: 1 = Black or African American (3) and Applicant Race: 2 = White (5)

OR 

Applicant Race: 1 is White (5) and Applicant Race: 2 =Black or African American (3)

THEN

Categorize as Black or African American
```

Otherwise if race 1 and race 2 equal =Black or African American (3)  it'll be a valid scenario

**Comment by kgudel** [comment]
Closed in #2740
